enterprise/backup/src/test/java/org/neo4j/backup/impl/BackupHelpOutput.java

```diff
@@ -64,8 +64,7 @@
         lines.add( "  --name=<graph.db-backup>                 Name of backup. If a backup with this" );
         lines.add( "                                           name already exists an incremental" );
         lines.add( "                                           backup will be attempted." );
-        lines.add( "  --from=<address>                         Host and port of Neo4j." );
-        lines.add( "                                           [default:localhost:6362]" );
+        lines.add( "  --from=<address>                         Host and port of Neo4j. [default:]" );
```

Comment:
Is that correct with just [default:] ?

Reply:
I will fix this up, because now it will be the default backup address. I will make it dependant on config though.